MyServlet/build.gradle

```diff
@@ -0,0 +1,36 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'org.akhikhl.gretty:gretty:+'
+    }
+}
+
+plugins {
+    id 'java'
+}
+apply plugin: 'war'
+apply plugin: 'org.akhikhl.gretty'
+group 'HelloServlet'
```

Comment:
Usually logical blocks are separated with one vertical space (between 12-13 and 14-15 rows)